--- activate	2017-07-11 18:11:55.000000000 -0400
+++ activate_new	2017-07-11 18:14:24.000000000 -0400
@@ -97,7 +97,7 @@
     if [ $("$_CONDA_DIR/conda" ..changeps1) = "1" ]; then
         if ! $(grep -q CONDA_DEFAULT_ENV <<<$PS1); then
             if ! $(grep -q "POWERLINE" <<<$PS1); then
-                export PS1="(${CONDA_DEFAULT_ENV}) $PS1"
+                export PS1="\$(if [[ \$? == 0 ]]; then echo \"\[\e[00;30m\]\"; else echo \"\[\e[00;31m\]\"; fi)(${CONDA_DEFAULT_ENV}) [\A] [\h \u]:\W \#\$\[\e[0m\] \[\e[33m\]"
             fi
         fi
     fi
